ambu

```diff
@@ -1,22 +1,25 @@
 #!/bin/bash
 
 MAKEMODE=$1
-CURRENTDIR=`pwd`
+CURRENTDIR=${2:-`pwd`}
 PROJECT=`basename $CURRENTDIR`
 PARENTDIR=`dirname $CURRENTDIR`
 BUILDDIR=$PARENTDIR/".build_"$PROJECT
 PROJECTNAME=$PROJECT".ino"
 
-usage()
-{
-cat << EOF
-usage: $0 [makefile target]
-(see Arduino Makefile for target details)
-EOF
+usage() {
+    echo "usage: $0 target [project dir]"
+    echo "(see Arduino Makefile for target details)"
```

Comment:
@Adman What does this mean? Where would I look for this target details? Could this message be more informational?